ui/components/multichain/pages/connections/connections.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import {
+  AlignItems,
+  BackgroundColor,
+  Display,
+  IconColor,
+  JustifyContent,
+  TextAlign,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import { DEFAULT_ROUTE } from '../../../../helpers/constants/routes';
+import { getURLHost } from '../../../../helpers/utils/util';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import {
+  getOriginOfCurrentTab,
+  getSubjectMetadata,
+} from '../../../../selectors';
+import {
+  AvatarFavicon,
+  AvatarFaviconSize,
+  Box,
+  ButtonIcon,
+  ButtonIconSize,
+  ButtonPrimary,
+  ButtonPrimarySize,
+  Icon,
+  IconName,
+  IconSize,
+  Text,
+} from '../../../component-library';
+import { Content, Footer, Header, Page } from '../page';
+import { NoConnectionContent } from './components/no-connection';
+
+export const Connections = () => {
+  const t = useI18nContext();
+  const history = useHistory();
+  const activeTabOrigin = useSelector(getOriginOfCurrentTab);
+  const subjectMetadata = useSelector(getSubjectMetadata);
```

Comment:
```suggestion
  const subjectMetadata: { [key: string]: any } = useSelector(getConnectedSitesList);
```
I think we want to use getConnectedSitesList here, because even non-dapps can have subjectMetadata.

Reply:
Nice catch. Updated here https://github.com/MetaMask/metamask-extension/pull/23070/commits/c3beebf5edf3465f04f9edbeae1b4a6b2860ff7c